desktop/e2e/tests/e2e.js

```diff
@@ -19,14 +21,13 @@ const driverConfig = new webdriver.Builder()
 	.usingServer( 'http://localhost:9515' )
 	.setChromeOptions( options )
 	.withCapabilities( {
-		chromeOptions: {
+		'goog:chromeOptions': {
```

Comment:
Docs: https://www.electronjs.org/docs/tutorial/using-selenium-and-webdriver#3-connect-to-chromedriver